stages/org.osbuild.systemd.unit.create

```diff
@@ -48,8 +47,6 @@ def main(tree, options):
     with open(f"{systemd_dir}/{filename}", "w", encoding="utf8") as f:
         config.write(f, space_around_delimiters=False)
 
-    subprocess.run(["systemd-analyze", "verify", f"{systemd_dir}/{filename}"], check=True)
```

Comment:
Fwiw, I think this is fine for the reasons you outlined. It does more than we need and this means it gets in the way instead of helping us. Having a strict "syntax check only" mode or something would help us (maybe worth a bugreport?).

Reply:
Yeah or feature request or something.